File: Java_Programming/Single.java
class Base //8 bytes
{
    public int i, j;

    public void fun()

    {
        System.out.println("Inside Base Fun");
    }

}

class Derived extends Base  //12 bytes
{
    public int x;
    public void gun()
    {
        System.out.println("Inside Derived Gun");
    }
}

class Single
{
    public static void main(String A []) 
    {
       Base bobj = new Base(); 
       Derived dobj = new Derived();

       bobj.fun();
       dobj.fun();
       dobj.gun();
    }
}

File: C++_Programming/Single.cpp
#include<iostream>

using namespace std;

class Base
{
    public:
    int i,j;

    void fun()
    {
        cout<<"Inside base fun.";
    }
};

class Derived : public Base

{
    public:
    int x;

    void gun()
    {
        cout<<"Inside Gun.";
    }
};
int main()
{
    Base bobj;
    Derived dobj;

    cout<<"Base class object: "<<sizeof(bobj)<<"\n";
    cout<<"Derived class object: "<<sizeof(dobj)<<"\n";

    return 0;

}